packages/vscode-dependencies-validation/src/commands.ts

```diff
@@ -1,38 +1,40 @@
-import { dirname } from "path";
-import type { DiagnosticCollection, OutputChannel } from "vscode";
-import { invokeNPMCommand } from "@sap-devx/npm-dependencies-validation";
-import { refreshDiagnostics } from "./diagnostics";
+import type { DiagnosticCollection, Uri } from "vscode";
+import { VscodeCommandsConfig, VscodeOutputChannel } from "./vscodeTypes";
+import { FIX_ALL_ISSUES_COMMAND } from "./constants";
+import { clearDiagnostics, fixDepsIssues } from "./util";
 
-type VscodeOutputChannel = Pick<
-  OutputChannel,
-  "append" | "show" | "appendLine"
->;
-
-export async function fixAllDepIssuesCommand(
+async function fixProjectDepsIssues(
   outputChannel: VscodeOutputChannel,
-  packageJsonPath: string,
-  dependencyIssuesDiagnosticCollection: DiagnosticCollection
+  diagnosticCollection: DiagnosticCollection,
+  uri: Uri
 ): Promise<void> {
+  // switched to outputchannel only in manual mode
   outputChannel.show(true);
+  await fixDepsIssues(uri, outputChannel);
+  clearDiagnostics(diagnosticCollection, uri);
```

Comment:
Again there is an assumption here that the `fixDepsIssues` succeeded

Reply:
If it fails it does not reach clearDiagnostics